runner/execer/execer.go

```diff
@@ -30,8 +30,7 @@ type Execer interface {
 type Process interface {
```

Comment:
What is the expected behavior of Wait & Abort.  Comment on what this interface represents and the expected behavior of methods is would be nice.

I assume Wait() is waiting for something to execute.  Should it potentially return an error then? What if Abort is called while we are Waiting?